### PR TITLE
FCBHDBP-501 Fix the way to update language translation with  priority 5

### PR DESCRIPTION
**Pull request by vichugofsl**
## Description
It has improved the way to handle the language translation records with priority 5. Also, it has replaced the method update with `updateCol`.

We have changed the way to create the update statement to add information about the old value that should update. E.g.

```sql
UPDATE language_translations SET priority = '0' WHERE language_source_id='194' AND language_translation_id='6414' AND id='162891'; -- prior=9
```

## NOTE

@bradflood It has updated the logic to store the language translation records with EthName. if the name in AltName matches the EthName, ignore the AltName and if a language with priority 0 has the EthName value, it should change that priority to 5. The above is only with language_translation_id = 8012.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-501

## How Do I QA This
- I have executed the following command on my local environment:

```shell
python3 load/UpdateDBPLanguageTranslation.py test
```
- and I got th
[log_update_languages_translation_ethame.log](https://github.com/faithcomesbyhearing/dbp-etl/files/10143650/log_update_languages_translation_ethame.log)
e following outcome:

### CODE REVIEW

File: load/UpdateDBPBibleFileTags.py [hunk 1]
```diff
@@ -121,12 +121,11 @@ def process(self, hashId, inputFileset):
 				if tagValue == None:
 					insertRows.append((thumbnailFilename, bibleFileId, 'thumbnail', 0))
 				else:
-					updateRows.append((thumbnailFilename, bibleFileId, tag))
+					updateRows.append(("value", thumbnailFilename, tagValue, bibleFileId, tag))
 
 		tableName = "bible_file_tags"
 		attrNames = ("value",)
-		self.dbOut.insert(tableName, ("file_id", "tag", "admin_only"), attrNames, insertRows)
```
Comment (bradflood) [inline load/UpdateDBPBibleFileTags.py:128]: This seems unrelated to language_translations. please remove. There is ticket DBP-499 to change updateCol when unrelated to language_translation

File: load/UpdateDBPLanguageTranslation.py [hunk 2]
```diff
@@ -85,26 +89,65 @@ def _updateOrInsertLangTranslationsLowerPriority(self, languageNames, languageSo
...
+                            updateRows.append(("priority", priority, oldPriority, languageSourceId, languageTranslationId, languageName, translationIdWithSameLangAndName))
+                            self.dbOut.updateCol(tableName, pkeyNames, updateRows)
                     else:
                         languageName = languageName.replace("'", "\\'")
                         attrNames = ("name", "priority")
                         pkeyNames = ("language_source_id", "language_translation_id")
                         insertRows.append((languageName, priority, languageSourceId, languageTranslationId))
                         self.dbOut.insert(tableName, pkeyNames, attrNames, insertRows)
 
+    def _updateOrInsertLangTranslationsEthNamePriority(self, languageName, languageSourceId, languageTranslationId, priority):
```
Comment (bradflood) [inline load/UpdateDBPLanguageTranslation.py:113]: rename to _updateOrInsertLangTranslationsEthName

**Review by bradflood — CHANGES_REQUESTED** [review]

File: load/UpdateDBPBibleFileTags.py (hunk 1 above)
Comment (vichugofsl) [reply load/UpdateDBPBibleFileTags.py]: @bradflood , It has been removed from this PR.

File: load/UpdateDBPLanguageTranslation.py (hunk 2 above)
Comment (vichugofsl) [reply load/UpdateDBPLanguageTranslation.py]: @bradflood , it has been changed as you can see in the following line:

https://github.com/faithcomesbyhearing/dbp-etl/pull/95/files#diff-d21d9d4001c386f3021901b139bfa9036284349ef9f6a39ccb598a08c7234787R113

**Review by bradflood — APPROVED** [review]
